Classes/ZedBaseHusk.uc

```diff
@@ -12,6 +12,10 @@ simulated function PostBeginPlay()
 {
     super.PostBeginPlay();
 
+    // and why TWI removed this feature...
+    if (Controller != none)
```

Comment:
Controller check is unnecessary here:

1. MyAmmo does not require Controller.
2. At this moment, the Controller is always set, unless a modder messed up something.
3. Even in the theoretical case where the Controller is none, you will get tons of warning messages below for accessing none MyAmmo.

Reply:
It was just a copy-cat from Monster's code. Can be removed ofc.